Dockerfile

```
@@ -7,7 +7,7 @@ WORKDIR /app
 COPY package*.json ./
 
 ## Install only the packages defined in the package-lock.json (faster than the normal npm install)
-RUN npm set progress=false && npm ci --production
+RUN npm set progress=false && npm ci --no-cache
```

Don't we rather want?
```suggestion
RUN npm set progress=false && npm ci --no-cache --production
```